kubetest/kops_test.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+)
+
+///
+
+/*
+Per @justinsb
+TODO @chrislovecnm
+If you wanted to write a test, you could split getRandomAWSZones into two functions - one that builds the map, the other that returns a list of zones with at least N members
```

Comment:
I just don't really see the point of these skipped test functions.  We're going to find out PDQ if it doesn't work...